CFC_DataCollector/recommender/utility_model_pipeline.py

```diff
@@ -0,0 +1,17 @@
+"""
+Construct user utility model or retrieve from database and update with
+augmented trips. Store in database and return the model.
+"""
+
+from user_utility_model import UserUtilityModel
+
+def build_user_model(user_id, augmented_trips):
+  model = UserUtilityModel.find_from_db(user_id)
+
+  if model:
+    model.update(augmented_trips)
+  else:
+    model = UserUtilityModel(user_id, augmented_trips)
```

Comment:
So this means that you need to add a step to calculate modelPrime somewhere (in the pipeline or hidden in the module).

Note also the comments around calculating the utility of the to-be-recommended trips (in the "improve trips pipeline") and the comment about getting trips in this pipeline.